examples/codesandbox/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "react",
+    "version": "1.0.0",
+    "description": "React example starter project",
+    "keywords": [
+      "react",
+      "starter"
+    ],
+    "main": "src/index.js",
+    "dependencies": {
+      "react": "16.13.1",
+      "react-dom": "16.13.1",
+      "react-scripts": "3.4.3",
+      "text-provider": "0.0.37"
+    },
+    "devDependencies": {
+      "typescript": "3.8.3"
```

Comment:
Why is typescript a devDependency ?

Reply:
Removed - this was added by default by Codesandbox.io